akka-samples/akka-docs-java-lambda/src/test/java/util/SocketUtils.java

```diff
@@ -0,0 +1,27 @@
+package util;
+
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import java.net.ServerSocket;
+import java.nio.channels.ServerSocketChannel;
+
+public class SocketUtils {
+
+  public static InetSocketAddress temporaryServerAddress(String hostname) {
```

Comment:
we already have this defined somewhere. I think akka-actor-tests and stream-tests have theirs